make/policies.cmake

```diff
@@ -44,8 +44,5 @@ cmake_policy(SET CMP0042 OLD)
 # XXX DrMem-i#1481: update to cmake 2.8.12's better handling of interface imports
 cmake_policy(SET CMP0022 OLD)
 
-# XXX i#1418: update to cmake 2.8.12's better handling of interface imports, for Qt/Win
```

Comment:
I'm confused as to why we got this warning in the first place: do you understand why?  I agree the CMP0020 description looks like it doesn't apply to us (I don't see qtmain or qtcore in our code) but I don't think drgui has changed at all so if this was raised back then you'd think the issue would remain.

Reply:
I haven't verified but older version may have warned if the policy wasn't set. From --help-policy CMP0020: "CMake version
3.10.2 warns when the policy is not set and uses OLD behavior.  Use
the cmake_policy command to set it to OLD or NEW explicitly."

So now I'm actually surprised I am not getting any warning by not setting it, sounds like you have to set it. I'd suggest since the warning is gone by not setting, this is probably the most recent behavior and I'd leave it like this.